compute/rn/release-information/release-notes-22-06-update2.adoc

```diff
@@ -0,0 +1,112 @@
+//
+// LINK
+
+toc::[]
+
+=== Enhancements
+// GithubIssue PCSUP-issue#
+
+// GH#39229
+==== HTTPS Proxy Support for Agentless Scanning
+
+* Agentless scanning now supports connections over an HTTPS proxy server.  If you use custom certificates for authentication, you can now configure custom certificates for the connection to Console when using https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/configure/configure-agentless-scanning[agentless scanning].
+
+// GH# 39754
+==== Support for Embedding a Defender in a CloudFormation Fargate Task
+* Prisma Cloud Compute now supports embedding a Defender to a CloudFormation Fargate task in the YAML format, in addition to the JSON format.
++
+You can use a full CloudFormation template that contains other objects in addition to the Fargate task to generate a protected Fargate task definition.
++
+Use the Console (Manage > Defenders > Deploy > Defenders) or the APIs (/api/22.06/defenders/fargate.yaml, /api/22.06/defenders/fargate.json) to complete the workflow.
+
+// GH#39262
+==== Support for More Orchestrators
+  
+** Google Kubernetes Engine (GKE) version 1.24.2 with containerd version 1.5.13
+
+** Elastic Kubernetes Service (EKS) version 1.23.7
+
+** Azure Kubernetes Service (AKS) version 1.24.3 running on Linux
+
+** AKS version 1.24.3 running on Windows
+
+** Lightweight Kubernetes (k3s) version v1.24.4+k3s1 with containerd v1.5.13-k3s1
+
+** Openshift version 4.11 with CRIO
+
+** Rancher Kubernetes Engine (RKE) version 1.24.4+rke2r1 with containerd 1.6.6-k3s1
+
+//GH#39872
+==== Name Update for Cloud Native Network Firewall (CNNF) 
+The Cloud Native Network Firewall (CNNF) is now renamed as Cloud Native Network Segmentation (CNNS). 
+
+
+=== Addressed Issues
+
+// GH#40449 PCSUP-10632
+* Fixed an issue that caused Defender to incorrectly report the Host OS as SLES15SP1 instead of SLES15.
+
+// GH#40673 #PCSUP-10988  #PCSUP-1940
+* Fixed an internal error that failed to refresh the vulnerability statistics under *Monitor > Vulnerabilities > Vulnerability Explorer*.
+
+// GH#39394	PCSUP-9241
+* Fixed an explicit deny error when sending a test alert to the AWS Security Hub integration. With this fix, the correct AWS product ARN for US and China regions are used.
+
+// GH#32310 PCSUP-10507
+* Fixed an issue for Defenders running on containerd/CRI-O nodes where the image had a mount point to the host filesystem and some parent directory of the mount point was a symlink, leading to host scanning.
+
+// GH#40515 PCSUP-10791
+Fixed an issue that prevented editing WAAS rules. On upgrade to 22.06, it was not possible to update or modify WAAS rules configured to protect the same port at multiple endpoints with different attributes, such as TLS, HTTP2, and gRPC.
+With this fix, such rules can now be modified.
+
+// GH#40616 PCSUP-10841
```

Comment:
@rcaballeromx this is SaaS only. I think we need to delete it in the on-prem RN here.

Reply:
@Pubs-MV I'll remove it and create a separate PR to add it to the admin guide for SAAS only. The sooner we document this limitation the better.